--- iio-sensor-proxy-1.0/src/drivers.h~	2015-05-22 16:16:20.000000000 +0200
+++ iio-sensor-proxy-1.0/src/drivers.h	2015-05-24 15:35:07.618426157 +0200
@@ -58,7 +58,7 @@
 	void     (*close)       (void);
 };
 
-inline gboolean
+static inline gboolean
 driver_discover (SensorDriver *driver,
 		 GUdevDevice  *device)
 {
@@ -69,7 +69,7 @@
 	return driver->discover (device);
 }
 
-inline gboolean
+static inline gboolean
 driver_open (SensorDriver       *driver,
 	     GUdevDevice        *device,
 	     ReadingsUpdateFunc  callback_func,
@@ -83,7 +83,7 @@
 	return driver->open (device, callback_func, user_data);
 }
 
-inline void
+static inline void
 driver_set_polling (SensorDriver *driver,
 		    gboolean      state)
 {
@@ -95,7 +95,7 @@
 	driver->set_polling (state);
 }
 
-inline void
+static inline void
 driver_close (SensorDriver *driver)
 {
 	g_return_if_fail (driver);
